### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by iLynette**
In this project we implemented the following;

- A Navigation bar.

- Three content sections:

1. Books list.

2. Add book form.

3. Contact info.

- Used Javascript to ensure that when the nav menu was clicked, it redirects you to the specific part of the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
+      </nav>
+    </header>
+    <section>
+      <p id="time"></p>
+    </section>
+    <section>
+      <div class="book-list">
+        <h1 class="heading">All Awesome Books</h1>
+        <div class="showBooks" id="showBooks"></div>
+      </div>
```
Comment (Lameck1) [inline index.html]: - [x] I suggest that we improve on this section that displays the list of added books. Our implementation as compared to the given wireframe lacks the alternating background for the list items. Please consider working on this. See screenshots below for comparison:

| Wireframe| Implementation |
| --- | --- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -1,29 +1,66 @@
...
+          Do you have any questions or do just want to say hello?<br />
+          You can reach out to us
+        </p>
+        <ul>
+          <li>completewebsite@gmail.com</li>
+          <li>Our phone number: 3632809218</li>
+          <li>Our address: 74 Kimathi street</li>
+        </ul>
+      </div>
+    </section>
```
Comment (Lameck1) [inline index.html]: - [x] There is nothing wrong with this section. It is just an emphasis that we ensure we stick the footer to the bottom section of the page as in the wireframes. Please work on this. Screenshots:

| Wireframe | Implementation |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -1,29 +1,66 @@
...
+    <header>
+      <nav>
+        <p>Awesome Books</p>
+        <ul class="link">
+          <li><a href="#" class="list-nav">List</a></li>
+          <li>|</li>
+          <li><a href="#" class="books-nav">Add new</a></li>
+          <li>|</li>
+          <li><a href="#" class="contact-nav">Contact</a></li>
+        </ul>
```
Comment (Lameck1) [inline index.html]: - [x] Last but not least, let's ensure that our navigation links have the `isactive` class to ensure that a link is styled differently from the rest when it is active. Active means it points us to what section of the App we are currently in.

  Looking at the wireframes we'll realize that the active links are denoted by changing their text color to blue. We can realize that by looking at the screenshots below and making the necessary adjustments:
  
  | Active links color from the wireframe | Active links don't change in the implementation|
  | --- | --- |
  | image image image | image |

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @iLynette & @Tufahel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Tufahel) [reply index.html]: Thanks for your review @Lameck1 . We have worked on that and solved it.

File: index.html (hunk 3 above)
Comment (Tufahel) [reply index.html]: Thanks for your review @Lameck1 . We have worked on that and solved it.

File: index.html (hunk 2 above)
Comment (Tufahel) [reply index.html]: Thanks for your review @Lameck1 . We have worked on that and solved it.

File: index.html [hunk 4]
```diff
@@ -1,29 +1,69 @@
...
             </label>
             <button type="submit" id="addBtn">Add</button>
-        </form>
-    </div>
-<script src="file.js"></script>
-</body>
-</html>
\ No newline at end of file
+            </form>
+        </div>
```
Comment (Mihndim2020) [inline index.html]: - [OPTIONAL] As requested by the previous code reviewer, which is also a requirement for this project and a best practice for a front end developer(`sticking to the design as much as possible`), please kindly implement alternating background color for the book list items and shown in the image below. 

alternatingBackground

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @iLynette, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**